common/src/main/java/com/wynntils/core/WynntilsMod.java

```diff
@@ -40,6 +41,7 @@ public static void init(String modVersion) {
         parseVersion(modVersion);
 
         WebManager.init();
+        ConfigManager.init();
```

Comment:
I'd say to init the `ConfigManager` first

Reply:
In case we ever support some form of cloud configs, `ConfigManager` would rely on `WebManager` being initialized

Reply:
Oh yeah, totally forgot about that